src/kalendar/dekad.py

```diff
@@ -110,7 +110,7 @@ def __add__(  # type: ignore[override]
             new_dekad = self._dekad_adjuster(self.dekad, other)
             return Dekad(year=new_year, dekad=new_dekad)
         else:
-            return self.todate() + other
+            return self.todate() + other  # type: ignore
```

Comment:
I think the issue with the type is because it should be a `datetime`, not `timedelta`